frontend/src/pages/RegisterInvestor.css

```
@@ -0,0 +1,375 @@
+  border: 1px solid var(--border);
+  border-radius: 6px;
+  padding: 40px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  text-align: left;
+}
+
+.register-investor__title {
+  font-family: 'Geologica', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geologica" (font-family-name-quotes)

(font-family-name-quotes)